ui/apps/platform/cypress/integration/listeningEndpoints/ListeningEndpoints.helpers.js

```diff
@@ -9,12 +9,12 @@ export function visitListeningEndpointsFromLeftNav() {
 
 export function selectFilterEntity(entity) {
     cy.get(selectors.entityDropdownToggle).click();
-    cy.get(`${selectors.entityDropdownMenuItems} button:contains("${entity}")`).click();
+    cy.get(`div${selectors.entityDropdownToggle} button:contains("${entity}")`).click();
```

Comment:
Am curious, is `div` needed?

Although I see attribute selector, at first glance it looks like possible typo (like lack of space) compared to many other template strings for multiple selectors.